packages/core/src/Popover/Overlay.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box } from '../Box/Box'
+
+const StyledOverlay = styled(Box)`
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  opacity: ${({ overlayOpacity }) => overlayOpacity};
+  cursor: pointer;
+  display: ${({ popoverOpen }) => (popoverOpen ? 'block' : 'none')};
+`
```

Comment:
I would prefer if we replaced this with the `Shadow(Overlay|Effect)` component so that it was easier to maintain consistency

Reply:
That'd be a nice future enhancement. I took this from the old popover to be sure the new one starts with the same styles.